### DISCUSSION TITLE
vits有关

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#471] [一般讨论]
突然想起来cqps有vits，在琢磨要不要开始玩起来，现在部署也不是3566小霸王上了
读wiki时看到
`
    一些预训练模型可以在 [CjangCjengh/TTSModels](https://github.com/CjangCjengh/TTSModels#vits) 下载（注意是下载 VITS 的，不是 HuBERT-VITS 或 W2V2-VITS）`
但是vits-simple-api现在其实能支持这些模型，这句是当时不支持所以写在这还是cqps没法调用其他格式的模型
（并不是FR，我估计我会懒得折腾，笑）

**Comment by Tsuk1ko** [comment]
因为我当时只处理了 vits 模型的，其他的按理来说也能用，但是要额外写逻辑，我也懒得（

https://github.com/Tsuk1ko/cq-picsearcher-bot/blob/fef175e5a55b4f12c31116510a5d2b8fe32aa02f/src/plugin/vits.mjs#L101-L113
